astro/src/content/docs/release-notes/index.mdx

```diff
@@ -16,6 +16,25 @@ import ReleaseNoteHeading from 'src/components/docs/release-notes/ReleaseNoteHea
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications)
 
+<ReleaseNoteHeading version="1.48.2" releaseDate="November 21th, 2023" />
+
+### Fixed
+* In version `1.48.1` the ability to page past 10,000 results on the Users and Entities search pages when using the Elasticsearch engine was added. See [Github Issue #494](https://github.com/FusionAuth/fusionauth-issues/issues/494). There are two sets of pagination controls on these pages: one above and one below the results. If you paged past 10,000 results and clicked the "Next" button on the bottom controls there would be an error and the page would not advance. 
+  * Resolves [GitHub Issue #2544](https://github.com/FusionAuth/fusionauth-issues/issues/2544)
+* In some cases when interacting with FusionAuth-hosted pages in an non-secure context (such as local development) it was possible to receive an error in the browser PublicKeyCredential is not defined. This may have disrupted some flows such as passwordless login. This has been fixed.
```

Comment:
Suggestion:

> In some cases when using with FusionAuth-hosted pages in an non-secure context, such as accessing FusionAuth on `localhost`, the `PublicKeyCredential` JavaScript API will not be available. This may cause an error on your JavaScript console `PublicKeyCredential is not defined`. This error kept the form on the page from correctly submitting.